plaidml/edsl/ffi.cc

```diff
@@ -987,6 +989,17 @@ plaidml_program* plaidml_program_evaluate(  //
         args[i].buffer = nullptr;
       }
     }
+
+    auto context = ret->program->module->getContext();
+    ret->program->dtypes.floatx =
+        pmlc::dialect::eltwise::ScalarType::get(context, static_cast<pmlc::util::DataType>(floatx))
+            .toStandard()
```

Comment:
We should avoid using standard types at this stage. At this stage, we should use the `ScalarType`.

It's only later when we lower from `tile` to `std` would we want to transition to using standard types.